src/movies.js

```diff
@@ -1,22 +1,78 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+  return moviesArray.map((movie) => {});
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+  if (moviesArray.length === 0) {
+    return 0;
+  }
+  const spielbergDramas = moviesArray.filter(
+    (movie) => movie.director === "Steven Spielberg" && movie.genre === "Drama"
```

Comment:
the property genre in the movies data is an Array of genres something like this 
`    genre: ['Crime', 'Drama'],` so what we need to use is the include methods for the array would be something like this 

```suggestion
    (movie) => movie.director === "Steven Spielberg" && movie.genre.includes('Drama')
```